src/main/java/com/netflix/imflibrary/st0377/header/J2KExtendedCapabilities.java

```diff
@@ -0,0 +1,34 @@
+package com.netflix.imflibrary.st0377.header;
+
+import com.netflix.imflibrary.MXFPropertyPopulator;
+import com.netflix.imflibrary.annotations.MXFProperty;
+import com.netflix.imflibrary.st0377.CompoundDataTypes;
+import com.netflix.imflibrary.utils.ByteArrayDataProvider;
+import com.netflix.imflibrary.utils.ByteProvider;
+
+import javax.annotation.concurrent.Immutable;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Object model corresponding to J2KExtendedCapabilities as defined in ISO/IEC 15444-1:2019 Annex A.5.2
+ */
+
+@Immutable
+public final class J2KExtendedCapabilities {
+
+    @MXFProperty(size=4) protected final Integer pCap = null;
+    @MXFProperty(size=0, depends=true) protected final CompoundDataTypes.MXFCollections.MXFCollection<Short> cCapi = null;
```

Comment:
Lets change this to `cCap`, as the _'i'_ in the spec document is just used to indicate index.